packages/yourloops/components/statistics/sensor-usage-stat.tsx

```diff
@@ -29,24 +29,29 @@ import { SimpleStat } from 'dumb'
 import { StatFormats } from 'dumb/dist/src/models/stats.model'
 import { t } from 'i18next'
 import Box from '@mui/material/Box'
+import { type VizDataUtil } from 'tidepool-viz'
+import { ensureNumeric } from 'dumb/dist/src/components/stats/stats.util'
 
 interface SensorUsageStatProp {
-  total: number
-  sensorUsage: number
+  dataUtil: VizDataUtil
 }
 
 export const SensorUsageStat: FunctionComponent<SensorUsageStatProp> = (prop) => {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Nice component good job buddy 😉